config/samples/azure/kustomization.yaml

```diff
@@ -1,2 +1,3 @@
 resources:
+- azure_openshift_route.yaml
```

Comment:
I'm fine with any name but I would really prefer the things to be consistent.
The other file in this directory is called `operator_v1alpha1_externaldns_detailed.yaml`. BTW it's the kubebuilder pattern for the samples.